crm-project/src/main/java/CRM/utils/Validations.java

```diff
@@ -144,6 +144,15 @@ public static void validateTwoIds(Long firstId, Long secondId){
         validate(secondId, Regex.ID.getRegex());
     }
 
+    public static void validateUpdateUserToBoard(Long firstId, Long secondId, Long permissionId){
+        validateTwoIds(firstId, secondId);
```

Comment:
Don't use this function, because then you'll need a "validateThreeIds" and "validateFourIds"
I don't think you need a function for all of that anyway, but if you really want to, just create a function:
``
public static void validateIDs(...ids){
for(Long id: ids) validate(id, Regex.ID.getRegex()
}
``